tests/ext/scope_manager/test_threalocal.py

```diff
@@ -19,44 +19,12 @@
 # THE SOFTWARE.
 
 from __future__ import absolute_import
-import mock
+from unittest import TestCase
```

Comment:
I think the file name should be `test_threadlocal.py` (missing a `d`?)

Reply:
Oh, my bad, yes ;)